static/sass/_snapcraft_market_screenshots.scss

```diff
@@ -1,79 +1,77 @@
 @mixin snapcraft-market-screenshots {
-  .p-screenshots-toolbar {
-    align-items: baseline;
-    display: flex;
-    justify-content: space-between;
+  .p-listing-images {
+    align-items: flex-start;
```

Comment:
This doesn't seem to change anything (?).

As far as I remember `align-items` is about aligning items orthogonally to flex direction (so vertically in this case) - which is not needed (until container is bigger then the items).

Reply:
Ah this is left over from when there was the larger image - it ensured subsequent images were aligned with the top. Will remove